test/lib/jdk/test/lib/format/Diff.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * Represents a possible difference between two objects.
+ */
+package jdk.test.lib.format;
+
+/**
+ * An abstraction representing formattable difference between two or more objects
+ */
+public interface Diff {
+
+    /**
+     * Default limits for formatters
+     */
+    public static class Defaults {
```

Comment:
I'd add a no-op private no-arg ctor to show that this class isn't expected to be instantiated.

Reply:
Got it, fixed in the 78ed79f38262fec1df3ffb7ba407dc792666d1fd.